Makefile.am

```
@@ -271,6 +271,8 @@ COVERAGE_OUTPUTS = --txt coverage/coverage.txt \
 	--cobertura coverage/coverage.cobertura.xml \
 	--html-details coverage/details.html/coverage.details.html \
 	--coveralls coverage/coverage.coveralls.json
+# See https://gcovr.com/en/stable/guide/gcov_parser.html#negative-hit-counts
+COVERAGE_FLAGS=--gcov-ignore-parse-errors=negative_hits.warn_once_per_file
 COVERAGE_GCOV_EXE=--gcov-executable /usr/bin/gcov
 COVERAGE_FILTERS=-e '.*Test\.cpp$$' \
```

We could be a bit more consistent with spacing to match the rest of the file:
```suggestion
COVERAGE_FLAGS = --gcov-ignore-parse-errors=negative_hits.warn_once_per_file
COVERAGE_GCOV_EXE = --gcov-executable /usr/bin/gcov
COVERAGE_FILTERS = -e '.*Test\.cpp$$' \
```